README.md

```diff
@@ -5,14 +5,37 @@ Just a simple pecan app
 
 1. clone the repo
 
-2. install pecan in virtual env: https://pecan.readthedocs.io/en/latest/installation.html
+2. cd flocx-market and run cmd:
+```
+pip install -r requirements.txt
+```
+3. migrate the database:
+```
+cd flocx_market/db/sqlalchemy
+```
+```
+alembic upgrade head
+```
+
+# List available offers
+1. First create a offer record in the marketplace database, offer table (from MySQL cmd line)
```

Comment:
It would be nice to have a Python script that would create the sample data by instantiating SqlAlchemy models (instead of requiring someone to enter raw SQL commands).

Reply:
Thank you! I will write a script to create sample data.